src/mount/readdata.h

```diff
@@ -211,11 +210,19 @@ struct ReadRecord {
 	std::atomic<uint8_t> refreshCounter = 0;
 	std::atomic<uint16_t> requestsNotDone = 0;
 	bool expired = false; //gMutex
+	std::atomic<bool> stopThread;
```

Comment:
Consider using std::jthread if you need this behavior.

Reply:
Tested using `jthread` but the current approach was behaving more optimal and stable on tests. Because the `stopThread` variable allows me an easier cleaning control over the thread even before the object destruction at least for the case I needed.